ui/component/or-dashboard-builder/src/widgets/gateway-widget.ts

```diff
@@ -187,6 +204,17 @@ export class GatewayWidget extends OrWidget {
         }
     }
 
+    /**
+     * Internal function to set the active tunnel.
+     * Navigates the user to the tunnel after updating. This can be disabled using the {@link silent} parameter.
+     */
+    protected _setActiveTunnel(tunnelInfo?: GatewayTunnelInfo, silent = false) {
```

Comment:
Don't see any usage where `silent=true` which should be the case for tcp tunnels

Reply:
The protected `_navigateToTunnel()` function will automatically catch / ignore TCP tunnels.
(as there is no web URL to navigate to)

Currently I'm using the `silent` parameter during the `firstUpdated()` lifecycle function on line 112.
During first initialization of the page, the active tunnel is fetched and stored in cache using `_setActiveTunnel()`.
We only want to "open the tunnel in a new tab" when the tunnel is created using the "Start" button.
Not during page creation, so that's where the `silent` parameter comes in handy. 😉